### ISSUE TITLE
top_prospects examples doesn't work

### ISSUE THREAD

**Issue by ForceConstant** [jldbc/pybaseball#284]
Trying below, I get "No tables found"

```
from pybaseball import top_prospects

# Get top pitching prospects for the Toronto Blue Jays
topProspects = top_prospects("bluejays", "pitchers")
```

**Comment by tjburch** [comment]
Looks like mlb changed the url found [here](https://github.com/jldbc/pybaseball/blob/master/pybaseball/top_prospects.py#L10) to use TeamId rather than team name. We probably need to update it.

Team format: https://www.mlb.com/prospects/stats?teamId=111

**Comment by tjburch** [comment]
Being addressed in #307

**Comment by tjburch** [comment]
Closing with the merge of #307
